libft/sources/strings/strings.h

```diff
@@ -3,16 +3,18 @@
 /*                                                        :::      ::::::::   */
 /*   strings.h                                          :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/08/13 20:55:01 by inikulin          #+#    #+#             */
-/*   Updated: 2024/09/22 22:35:17 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/09 16:11:55 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef STRINGS_H
 # define STRINGS_H
 
+#include "strings_internal.h"
```

Comment:
This means all declarations from strings_internal.h are included into strings.h and hence into libft.h. This is undesired. I suggest including strings_internal only into ft_split_buf.c and ft_split_buf_utils.c. Including strings.h into strings_internal.h is fine, not the opposite.

Reply:
Make sense, thanks